platform/src/main/java/org/hillview/storage/CassandraConnectionInfo.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2020 VMware Inc. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.hillview.storage;
+
+public class CassandraConnectionInfo extends JdbcConnectionInformation {
+    static final long serialVersionUID = 1;
+
+    // Port for establising probe connection
+    public int jmxPort;
+    public String cassandraRootDir;
+
+    public CassandraConnectionInfo (String cassandraRootDir, String host, int jmxPort,
+        int nativePort, String keyspace, String table, String user, String password){
+        this.cassandraRootDir = cassandraRootDir;
```

Comment:
Do you want to add a constructor for JdbcConnectionInformation too? It wasn't needed so far, because that class is now read using serialization from json.